src/api/main.py

```
@@ -0,0 +1,234 @@
+        # Check component status
+        components: Dict[str, str] = {
+            "api": "healthy",
+        }
+
+        # Try to check runtime
+        try:
+            from src.api.dependencies import get_runtime
+
+            runtime = get_runtime()
+            components["runtime"] = runtime.state.value
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)